health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/downloadTemplate.js

```
@@ -0,0 +1,369 @@
+    const generatePaths = (data, path = [], allDatas) => {
+        const newPath = [...path, data.code];
+        allDatas.push(newPath);
+        if (data.children && data.children.length > 0) {
+            for (let child of data.children) {
+                generatePaths(child, newPath, allDatas);
+            }
+        }
+    };
+    const getAllDatas = (allBoundaries) => {
+        console.log("in the function", allBoundaries);
+        let allDatas = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'allDatas' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)